app/views/handlers/question.py

```diff
@@ -113,12 +113,42 @@ def get_next_location_url(self):
             self._return_to_block_id,
         )
 
+    # pylint: disable=protected-access
```

Comment:
Similar to prev comment, if this is now being used non-privately, consider making public or creating a public alias to safely call it.

Reply:
This code now reverted to its original state.